tools/api-client/python/lib/bmutils.py

```diff
@@ -94,11 +94,11 @@ def wrap_load_completed_games(self):
       raise ValueError, "Failed to call loadCompletedGames, got: " + retval
     return self._wrap_game_list_data(retval.data)
 
-  def wrap_create_game(self, pbutton, obutton='', opponent=''):
-    retval = self.create_game(pbutton, obutton, opponent)
+  def wrap_create_game(self, pbutton, obutton='', player='', opponent='', description=''):
+    retval = self.create_game(pbutton, obutton, player, opponent, description)
     if not retval.status == 'ok':
-      raise ValueError, "Failed to call createGame, got: " + retval
-    return retval.data
+      raise ValueError, "Failed to call createGame, got: " + retval.message
+    return retval
```

Comment:
Huh, sorry, i didn't notice this change yesterday (from returning `retval.data` to returning `retval`).  That's going to change the expectations of callers if any, isn't it?  And i don't think there's anything interesting in `retval.message` on success.

What does this change get you that you want?